packages/flutter/lib/src/services/binding.dart

```diff
@@ -415,6 +416,12 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
   void setSystemUiChangeCallback(SystemUiChangeCallback? callback) {
     _systemUiChangeCallback = callback;
   }
+
+  /// Alert the engine that the binding is registered. This instructs the engine to
+  /// register its top level window handler on Windows.
```

Comment:
Could we mention something like "This signals that the app is able to process `System.requestAppExit` messages from the engine."?